tests/test_data/core/core__count_observation_lab_month.txt

```diff
@@ -1,14 +1,14 @@
 (10, None, '34533-0', 'Urine smell ammoniacal (finding)', None)
-(10, None, None, 'Brown color (qualifier value)', 'AMB')
-(10, None, '5778-6', 'Brown color (qualifier value)', 'AMB')
+(10, None, None, 'Brown color (qualifier value)', 'ambulatory')
+(10, None, '5778-6', 'Brown color (qualifier value)', 'ambulatory')
```

Comment:
This was consolodating on the code display value versus the actual code value - this i don't think is being looked at often, but this was one of the older vintage of count table we had.